components/linear_app/sources/issue-created-instant/issue-created-instant.mjs

```diff
@@ -7,7 +7,7 @@ export default {
   name: "New Created Issue (Instant)",
   description: "Emit new event when a new issue is created. See the docs [here](https://developers.linear.app/docs/graphql/webhooks)",
   type: "source",
-  version: "0.1.1",
+  version: "0.1.2",
```

Comment:
Since there is a [new functionality with backwards compatibility](https://pipedream.com/docs/components/guidelines/#versioning), bump minor version

```suggestion
  version: "0.2.0",
```